<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>MainWindow</class>
 <widget class="QMainWindow" name="MainWindow">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>1176</width>
    <height>596</height>
   </rect>
  </property>
  <property name="maximumSize">
   <size>
    <width>2000</width>
    <height>2000</height>
   </size>
  </property>
  <property name="windowTitle">
   <string>Cars Table</string>
  </property>
  <property name="styleSheet">
   <string notr="true">background-color: rgb(144, 144, 216);</string>
  </property>
  <widget class="QWidget" name="centralwidget">
   <widget class="QTableWidget" name="tableWidget">
    <property name="geometry">
     <rect>
      <x>10</x>
      <y>20</y>
      <width>951</width>
      <height>471</height>
     </rect>
    </property>
    <property name="styleSheet">
     <string notr="true">QTableWidget
{
	border: 2px solid rgb(77, 77, 115);
	background-color: rgb(255, 255, 255);
}</string>
    </property>
    <property name="verticalScrollBarPolicy">
     <enum>Qt::ScrollBarAsNeeded</enum>
    </property>
    <property name="horizontalScrollBarPolicy">
     <enum>Qt::ScrollBarAlwaysOff</enum>
    </property>
    <property name="selectionMode">
     <enum>QAbstractItemView::ExtendedSelection</enum>
    </property>
    <column>
     <property name="text">
      <string>NUM</string>
     </property>
    </column>
    <column>
     <property name="text">
      <string>Производитель</string>
     </property>
    </column>
    <column>
     <property name="text">
      <string>Марка</string>
     </property>
    </column>
    <column>
     <property name="text">
      <string>Страна</string>
     </property>
    </column>
    <column>
     <property name="text">
      <string>Год выпуска</string>
     </property>
    </column>
    <column>
     <property name="text">
      <string>Объем двигателя (см3)</string>
     </property>
    </column>
    <column>
     <property name="text">
      <string>Коробка передачи</string>
     </property>
    </column>
    <column>
     <property name="text">
      <string>Пробег (км)</string>
     </property>
    </column>
    <column>
     <property name="text">
      <string>Цена (в руб)</string>
     </property>
    </column>
   </widget>
   <widget class="QPushButton" name="add_button">
    <property name="geometry">
     <rect>
      <x>970</x>
      <y>60</y>
      <width>191</width>
      <height>23</height>
     </rect>
    </property>
    <property name="font">
     <font>
      <pointsize>14</pointsize>
     </font>
    </property>
    <property name="styleSheet">
     <string notr="true">QPushButton
{
	background-color: rgb(77, 77, 115);
	border: 2px solid white;
	color: white
}
QPushButton:pressed
{
	background-color: rgb(200, 200, 0);
}</string>
    </property>
    <property name="text">
     <string>Добавить</string>
    </property>
   </widget>
   <widget class="QPushButton" name="delete_button">
    <property name="geometry">
     <rect>
      <x>970</x>
      <y>100</y>
      <width>191</width>
      <height>23</height>
     </rect>
    </property>
    <property name="font">
     <font>
      <pointsize>14</pointsize>
     </font>
    </property>
    <property name="styleSheet">
     <string notr="true">QPushButton
{
	background-color: rgb(77, 77, 115);
	border: 2px solid white;
	color: white
}
QPushButton:pressed
{
	background-color: rgb(200, 200, 0);
}</string>
    </property>
    <property name="text">
     <string>Удалить (NUM)</string>
    </property>
   </widget>
   <widget class="QPushButton" name="reload_button">
    <property name="geometry">
     <rect>
      <x>970</x>
      <y>20</y>
      <width>191</width>
      <height>23</height>
     </rect>
    </property>
    <property name="font">
     <font>
      <pointsize>14</pointsize>
     </font>
    </property>
    <property name="styleSheet">
     <string notr="true">QPushButton
{
	background-color: rgb(77, 77, 115);
	border: 2px solid white;
	color: white
}
QPushButton:pressed
{
	background-color: rgb(200, 200, 0);
}</string>
    </property>
    <property name="text">
     <string>Обновить</string>
    </property>
   </widget>
   <widget class="QComboBox" name="comboBox_creator">
    <property name="geometry">
     <rect>
      <x>970</x>
      <y>230</y>
      <width>191</width>
      <height>22</height>
     </rect>
    </property>
    <property name="styleSheet">
     <string notr="true">background-color: rgb(255, 255, 255);</string>
    </property>
    <item>
     <property name="text">
      <string>BMW</string>
     </property>
    </item>
    <item>
     <property name="text">
      <string>VAG</string>
     </property>
    </item>
    <item>
     <property name="text">
      <string>Toyota</string>
     </property>
    </item>
    <item>
     <property name="text">
      <string>Honda</string>
     </property>
    </item>
    <item>
     <property name="text">
      <string>Renault</string>
     </property>
    </item>
    <item>
     <property name="text">
      <string>Nissan</string>
     </property>
    </item>
    <item>
     <property name="text">
      <string>Mitsubishi</string>
     </property>
    </item>
    <item>
     <property name="text">
      <string>Ford</string>
     </property>
    </item>
   </widget>
   <widget class="QComboBox" name="comboBox_country">
    <property name="geometry">
     <rect>
      <x>970</x>
      <y>310</y>
      <width>191</width>
      <height>22</height>
     </rect>
    </property>
    <property name="styleSheet">
     <string notr="true">background-color: rgb(255, 255, 255);</string>
    </property>
    <item>
     <property name="text">
      <string>США</string>
     </property>
    </item>
    <item>
     <property name="text">
      <string>Россия</string>
     </property>
    </item>
    <item>
     <property name="text">
      <string>Германия</string>
     </property>
    </item>
    <item>
     <property name="text">
      <string>Франция</string>
     </property>
    </item>
    <item>
     <property name="text">
      <string>Италия</string>
     </property>
    </item>
    <item>
     <property name="text">
      <string>Испания</string>
     </property>
    </item>
    <item>
     <property name="text">
      <string>Польша</string>
     </property>
    </item>
   </widget>
   <widget class="QLineEdit" name="lineEdit_probeg">
    <property name="geometry">
     <rect>
      <x>970</x>
      <y>390</y>
      <width>191</width>
      <height>22</height>
     </rect>
    </property>
    <property name="styleSheet">
     <string notr="true">background-color: rgb(255, 255, 255);</string>
    </property>
   </widget>
   <widget class="QLineEdit" name="lineEdit_year">
    <property name="geometry">
     <rect>
      <x>970</x>
      <y>470</y>
      <width>191</width>
      <height>22</height>
     </rect>
    </property>
    <property name="styleSheet">
     <string notr="true">background-color: rgb(255, 255, 255);</string>
    </property>
   </widget>
   <widget class="QPushButton" name="cancel_button">
    <property name="geometry">
     <rect>
      <x>490</x>
      <y>510</y>
      <width>171</width>
      <height>51</height>
     </rect>
    </property>
    <property name="font">
     <font>
      <family>Times New Roman</family>
      <pointsize>20</pointsize>
     </font>
    </property>
    <property name="styleSheet">
     <string notr="true">QPushButton
{
	background-color: rgb(77, 77, 115);
	border: 2px solid white;
	color: white
}
QPushButton:pressed
{
	background-color: rgb(200, 200, 0);
}</string>
    </property>
    <property name="text">
     <string>Закрыть</string>
    </property>
   </widget>
   <widget class="QPushButton" name="creator_button">
    <property name="geometry">
     <rect>
      <x>970</x>
      <y>200</y>
      <width>191</width>
      <height>23</height>
     </rect>
    </property>
    <property name="font">
     <font>
      <pointsize>14</pointsize>
     </font>
    </property>
    <property name="styleSheet">
     <string notr="true">QPushButton
{
	background-color: rgb(77, 77, 115);
	border: 2px solid white;
	color: white
}
QPushButton:pressed
{
	background-color: rgb(200, 200, 0);
}</string>
    </property>
    <property name="text">
     <string>По производителю</string>
    </property>
   </widget>
   <widget class="QPushButton" name="country_button">
    <property name="geometry">
     <rect>
      <x>970</x>
      <y>280</y>
      <width>191</width>
      <height>23</height>
     </rect>
    </property>
    <property name="font">
     <font>
      <pointsize>14</pointsize>
     </font>
    </property>
    <property name="styleSheet">
     <string notr="true">QPushButton
{
	background-color: rgb(77, 77, 115);
	border: 2px solid white;
	color: white
}
QPushButton:pressed
{
	background-color: rgb(200, 200, 0);
}</string>
    </property>
    <property name="text">
     <string>По стране</string>
    </property>
   </widget>
   <widget class="QPushButton" name="probeg_button">
    <property name="geometry">
     <rect>
      <x>970</x>
      <y>360</y>
      <width>191</width>
      <height>23</height>
     </rect>
    </property>
    <property name="font">
     <font>
      <pointsize>14</pointsize>
     </font>
    </property>
    <property name="styleSheet">
     <string notr="true">QPushButton
{
	background-color: rgb(77, 77, 115);
	border: 2px solid white;
	color: white
}
QPushButton:pressed
{
	background-color: rgb(200, 200, 0);
}</string>
    </property>
    <property name="text">
     <string>По пробегу</string>
    </property>
   </widget>
   <widget class="QPushButton" name="year_button">
    <property name="geometry">
     <rect>
      <x>970</x>
      <y>440</y>
      <width>191</width>
      <height>23</height>
     </rect>
    </property>
    <property name="font">
     <font>
      <pointsize>14</pointsize>
     </font>
    </property>
    <property name="styleSheet">
     <string notr="true">QPushButton
{
	background-color: rgb(77, 77, 115);
	border: 2px solid white;
	color: white
}
QPushButton:pressed
{
	background-color: rgb(200, 200, 0);
}</string>
    </property>
    <property name="text">
     <string>По году</string>
    </property>
   </widget>
   <widget class="QLineEdit" name="lineEdit">
    <property name="geometry">
     <rect>
      <x>972</x>
      <y>140</y>
      <width>191</width>
      <height>22</height>
     </rect>
    </property>
    <property name="styleSheet">
     <string notr="true">background-color: rgb(255, 255, 255);</string>
    </property>
   </widget>
  </widget>
  <widget class="QStatusBar" name="statusbar"/>
 </widget>
 <resources/>
 <connections/>
</ui>
